utils/common/src/main/kotlin/Extensions.kt

```diff
@@ -464,3 +464,12 @@ fun Throwable.collectMessages(): List<String> {
  * Recursively collect the messages of this [Throwable] and all its causes and join them to a single [String].
  */
 fun Throwable.collectMessagesAsString() = collectMessages().joinToString("\nCaused by: ")
+
+/**
+ * Retrieve query parameters specified by its [key].
```

Comment:
You should probably document that this returns an empty list (instead of e.g. null) in case of an absent key.

Reply:
But actually, we probably *do* want to distinguish between a key being absent, and its value being empty.

Reply:
I changed this method, so it can be used in a more generic way. I think your comment is obsolete in this new version.